plugins/flytekit-kf-pytorch/flytekitplugins/kfpytorch/task.py

```diff
@@ -427,6 +429,14 @@ def execute(self, **kwargs) -> Any:
 
         return exception_scopes.user_entry_point(self._execute)(**kwargs)
 
+    def _convert_run_policy(self, run_policy: RunPolicy) -> kubeflow_common.RunPolicy:
```

Comment:
This method is exactly the same as in `PytorchFunctionTask`. It also doesn't need access to `self`. Could you please remove the duplication by removing the method from the classes and instead turn it into a standalone function called e.g. `_convert_run_policy_to_flyte_idl`?

Reply:
Addressed